$OpenBSD$

Add the KeyMap Unzip instance required by newer semialign/these versions.

--- src/Data/Aeson/KeyMap.hs.orig
+++ src/Data/Aeson/KeyMap.hs
@@ -100,6 +100,7 @@
 import Prelude ((.), ($))
 import Prelude (Functor(fmap), Monad(..))
 import Prelude (Show, showsPrec, showParen, shows, showString)
+import Prelude (fst, snd)
 
 import Control.Applicative (Applicative)
 import Control.DeepSeq (NFData(..))
@@ -351,7 +352,6 @@
 
 import Data.List (sortBy)
 import Data.Ord (comparing)
-import Prelude (fst)
 
 -- | A map from JSON key type 'Key' to 'v'.
 newtype KeyMap v = KeyMap { unKeyMap :: HashMap Key v }
@@ -693,6 +693,9 @@
 instance SA.Zip KeyMap where
     zipWith = intersectionWith
 
+
+instance SA.Unzip KeyMap where
+    unzip (KeyMap m) = (KeyMap (fmap fst m), KeyMap (fmap snd m))
 instance SAI.ZipWithIndex Key KeyMap where
     izipWith = intersectionWithKey
 
